packages/grant-explorer/src/features/api/utils.ts

```diff
@@ -616,6 +616,10 @@ export const pinToIPFS = (obj: IPFSObject) => {
 };
 
 export const getDaysLeft = (epochTime: number) => {
```

Comment:
`epochTime` is usually use to name the initial time in 1970, shall we maybe call it just `time` or `targetTime`